colorBackground.js

```diff
@@ -0,0 +1,17 @@
+var usedColors = {};
+
+function isUsed(randomColor) {
+  return usedColors[randomColor] === true;
+}
+
+function colorBackground() {
+  var randomColor = '#' + Math.floor(Math.random() * 16777215).toString(16);
```

Comment:
can you separate the concern of getting a random color and changing the background